app/models/ncr/work_order.rb

```diff
@@ -120,11 +122,21 @@ def current_approver_email_address
         self.currently_awaiting_approvers.first.email_address
       elsif self.approving_official
         self.approving_official.email_address
+      elsif self.emergency and self.approvers.empty?
+        EMERGENCY_APPROVER_EMAIL
```

Comment:
I would expect it to return `nil` in this case...if this is only meant for display purposes, I would move it to a helper or a Decorator.